modules/mono/editor/GodotTools/GodotTools/BottomPanel.cs

```diff
@@ -24,48 +24,50 @@ public class BottomPanel : VBoxContainer
         private Button errorsBtn;
         private Button viewLogBtn;
 
-        private void _UpdateBuildTabsList()
+        private void _UpdateBuildTab(int index, int? currentTab)
```

Comment:
Would replacing `int?` with `int` and using a special value such as `-1` be appropriate here?

Reply:
I like that `int?` makes it explicit in the type that it must be checked against an invalid value. Maybe `-1` is faster though which would be a valid reason to change it.
However, I plan to get rid of the build tabs soon (to have a single build page) so it would be a waste of time to bother with this.

Reply:
>so it would be a waste of time to bother with this.

Which makes me wonder why I even bothered to refactor this method in the first place.